.ci/test.sh

```diff
@@ -9,5 +9,19 @@ cabal --write-ghc-environment-files=always new-build all
 sed "s/^ghci/ghc -fno-code/" clash-dev > clash-dev-test
 sh clash-dev-test
 
+# Check whether version numbers in snap / clash-{prelude,lib,ghc} are the same
+cabal_files="clash-prelude/clash-prelude.cabal clash-lib/clash-lib.cabal clash-ghc/clash-ghc.cabal"
+snapcraft_file="bindist/linux/snap/snap/snapcraft.yaml"
+versions=$(grep "^[vV]ersion" $cabal_files $snapcraft_file | grep -Eo '[0-9]+\.[0-9]+' | tr ' ' '\n')
+
+if [[ $(echo $versions | tr ' ' '\n' | wc -l) == 4 ]]; then
+    if [[ $(echo $versions | tr ' ' '\n' | uniq | wc -l) != 1 ]]; then
+        echo "Expected all distributions to have the same version number. Found: $versions"
+    fi
```

Comment:
If we're building a tag maybe we should check that the version is equal to tag (stored in `CI_COMMIT_TAG`)

Reply:
Good idea!

Reply:
Ooooof, I also forgot to include `exit 1`s :sweat: